packages/documentsearch/style/base.css

```diff
@@ -34,11 +34,16 @@
   outline: 0;
 }
 
-.jp-DocumentSearch-overlay button:hover {
+.jp-DocumentSearch-overlay button.jp-DocumentSearch-button-wrapper:disabled {
```

Comment:
```suggestion
.jp-DocumentSearch-button-wrapper:disabled > .jp-DocumentSearch-button-content {
```

Reply:
> I don't see the pointer cursor on the prev/next buttons in Firefox ESR, and the `cursor: not-allowed` rule doesn't seem to change the cursor to the 🚫 cursor as your animation shows.

@JasonWeill what it we change the selector to the one proposed above? The hand cursor gets set on `.jp-DocumentSearch-button-content` so I think we should be overriding it on the same level.